src/net/java/sip/communicator/impl/hid/HIDServiceImpl.java

```diff
@@ -77,19 +77,19 @@ protected HIDServiceImpl()
      */
     public void keyPress(int keycode)
     {
-        if(OSUtils.IS_WINDOWS || OSUtils.IS_MAC)
-        {
+        //if(OSUtils.IS_WINDOWS || OSUtils.IS_MAC)
```

Comment:
I decided to comment this code because if we work with Ctrl, Shift and Alt like with modifiers only we can't handle hotkeys like Ctrl+Shif+Left(Right) etc

Reply:
Did you test this on all platforms, i.e. Linux, Mac and Windows?
If the code isn't needed anymore, remove it instead of leaving it commente.d